packages/cli-kit/src/api/admin.ts

```diff
@@ -20,36 +20,15 @@ const UnknownError = () => {
 }
 
 export async function request<T>(query: RequestDocument, session: AdminSession, variables?: Variables): Promise<T> {
-  const version = await fetchApiVersion(session)
-  const url = adminUrl(session.storeFqdn, version)
-  const headers = await buildHeaders(session.token)
-  debug(`
-Sending Admin GraphQL request:
-${query}
-
-With variables:
-${variables ? JSON.stringify(variables, null, 2) : ''}
-
-And headers:
-${sanitizedHeadersOutput(headers)}
-`)
-  try {
+  const api = 'Admin'
+  return handlingErrors(api, async () => {
```

Comment:
Any reason not to do `handlingErrors('Admin', ....` directly?

Reply:
It's used further down in line 28